### ISSUE TITLE
lib/libstdc++.so.6 causing problem in Arch Linux after recent updates

### ISSUE THREAD

**Issue by escuta** [ossia/score#1618]
After a recent system update on a Pi with Aarch64 and using the current Aarch64 Score release, running score headless causes problems with: 

```
export QT_QPA_PLATFORM=eglfs
./ossia_score-3.3.2/ossia-score-eglfs --no-gui myscorefile.score
```

It gives gives this error:

~/ossia_score-3.3.2/lib/libstdc++.so.6: version `GLIBCXX_3.4.32' not found 

libstdc++.so.6 seems to be out of date. Deleting/disabling it, and using the system version, solves the problem.

**Comment by jcelerier** [comment]
oh ! that's strange, usually older libstdc++ works on new systems. Right now it's building on debian bookworm for the raspberry.. i'll investigate

**Comment by jcelerier** [comment]
hmm on fedora 41 aarch64 with libstdc++ 14.2.1 it works... it warrants further investigation :) is the arch system up to date ? is it a 'standard' arch linux ARM setup ? I'll try to set up a pi with one tonight

**Comment by jcelerier** [comment]
I added a part to the launch script that detects and deletes existing libstdc++.so.6 if on non-debian distro

**Comment by escuta** [comment]
Yes, up to date. If I run "strings ~/ossia_score-3.3.2/lib/libstdc++.so.6", it prints the following (among other things) but doesn't include GLIBCXX_3.4.32

```
GLIBCXX_3.4.19
GLIBCXX_3.4.20
GLIBCXX_3.4.21
GLIBCXX_3.4.22
GLIBCXX_3.4.23
GLIBCXX_3.4.24
GLIBCXX_3.4.25
GLIBCXX_3.4.26
GLIBCXX_3.4.27
GLIBCXX_3.4.28
GLIBCXX_3.4.29
GLIBCXX_3.4.30
```

If I do the same with /usr/lib/libstdc++.so.6 , it prints up to GLIBCXX_3.4.33

**Comment by jcelerier** [comment]
ah, I think I understand - likely ossia loads libstdc++ and then also loads the GPU driver which links against the system LLVM which may have been updated to use newer libstdc++ than what's in debian. 
The fix I pushed should fix that as it will remove libstdc++.so.6 when on ArchLinux, I'll tag you to try it when it's done !

**Comment by jcelerier** [comment]
Heya ! could you try this version ?
https://github.com/ossia/score/releases/download/continuous/ossia.score-master-rpi-aarch64.tar.gz 

It will remove the libstdc++.so.6 if it's on a more recent distro

**Comment by jcelerier** [comment]
heya ! can you try with [3.4.0](https://github.com/ossia/score/releases/tag/v3.4.0) ? I think it's definitely fixed

**Comment by jcelerier** [comment]
closing - @thibaudk told me he had tried at some point
